src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -1,21 +1,28 @@
-import React, { useRef, forwardRef } from "react";
+import React, { useRef, useCallback, forwardRef } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
 import ExpandCollapse from "../../ExpandCollapse/ExpandCollapse";
 
-const AccordionItem = forwardRef(({ children, title, className, iconSize, id, open, onClick }, ref) => {
-  const componentRef = useRef(null);
-  const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+const AccordionItem = forwardRef(
```

Comment:
AccordionItem is an exposed component - this is a breaking change, please change naming so it would be compatible to previous versions

Reply:
This is not a breaking change, as onClick was an internal param (Passed by Accordion).
And wasn't documented in AccordionItem.
Even if you passed it, It got override by Accordion param.

Reply:
AccordionItem is exported in the published components app - and if someone has used it , it will break it